src/js/components/common/button/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.css';
+
+class Button extends Component {
+  render() {
+    return (
+      <button
+        className='button'
+        onClick={this.props.action}>
+        {this.props.name}
```

Comment:
I would suggest use property name "text" fot this case. Button already has build in prop name and this may cause problems in future